src/AppEntry.js

```diff
@@ -57,14 +56,17 @@ const AppEntry = ({ useLogger, connectToDevTools }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [client]);
 
-    return <IntlProvider locale={navigator.language.slice(0, 2)} messages={messages} onError={console.log}>
+    return <IntlProvider locale={navigator.language.slice(0, 2)} messages={messages} onError={() => undefined}>
```

Comment:
We should use the `console.log` to see which messages are missing
```suggestion
    return <IntlProvider locale={navigator.language.slice(0, 2)} messages={messages} onError={console.log}>
```